llm/llm_functions.py

```diff
@@ -789,6 +787,10 @@ def get_or_create_embeddings(bill_details: BillDetails, emb_api: OpenAIEmbedding
             chunk_size = 2000,
             chunk_overlap = 200
         )
+
```

Comment:
Can you add an inline comment explaining why this is needede?

Reply:
Have added explanation